screen/features/src/main/java/com/ivy/features/FeaturesUiState.kt

```diff
@@ -3,11 +3,16 @@ package com.ivy.features
 import kotlinx.collections.immutable.ImmutableList
 
 data class FeaturesUiState(
-    val features: ImmutableList<FeatureUi>,
+    val features: ImmutableList<Feature>,
 )
 
-data class FeatureUi(
+data class FeatureItem(
+    val key: String,
     val name: String,
     val enabled: Boolean,
     val description: String?,
-)
+) : Feature
+
+data class FeatureHeader(val name: String) : Feature
+
+interface Feature
```

Comment:
1. Make a `sealed interface`
2. Rename to `FeatureItemViewState`

Reply:
Done